app/models/ingredient_request.rb

```diff
@@ -0,0 +1,4 @@
+class IngredientRequest < ApplicationRecord
+  belongs_to :ingredient
```

Comment:
Or we leave the ingredients as they are (one ingredient belogs to a dish) or we need a join table for ingredients and dishes